### DISCUSSION TITLE
Mail Template "Joomla! Extension Update: Autoupdate Results": Place holder ignored

### DISCUSSION THREAD

**Discussion by Rolli1962** [akeeba/panopticon#289] [General]
Have created the currently missing mail template and let done a autoupdate.
Mail was send, but placeholder not replaced with content. Only SITE_NAME and SITE_URL was filled.
Info-Line of Template:
The extension [EXTENSION_NAME] version [NEW_VERSION] will now be installed on your site “[[SITE_NAME]](https://github.com/akeeba/panopticon/discussions/[SITE_URL])”. Your site is currently running the older version [OLD_VERSION].

Currently active panopticon version Alpha Version 1.0.4-dev202311081912

**Comment by nikosdion** [comment]
There are no variables like the ones you describe and it's totally not how this particular email template works. You need to [read the documentation](https://github.com/akeeba/panopticon/wiki/Mail-Templates#joomla-extension-update-autoupdate-results).

**Comment by Rolli1962** [comment]
Thanks, didn't know the doku before. I had activated the auto-update for patch-versions, and got only one mail with autoupdate results. Have set [RENDERED_HTML] and [RENDERED_TEXT] in the template now. Would be work in the future ...
Best Roland
